client/src/components/footer/footer.js

```diff
@@ -14,7 +14,7 @@ import {
   onNextPageChangeTSize,
 } from "./edit/editPage";
 import { Panel as ColorPickerPanel } from "rc-color-picker";
-import { Sections } from "../../constants/index";
+//import { Sections } from "../../constants/index";
```

Comment:
You can delete unused code, we can always look back at older commits if we need to get this code back